installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -1313,6 +1313,9 @@ protected String getExistingSaaSBoostStackName() {
         try {
             GetParameterResponse response = ssm.getParameter(request -> request.name("/saas-boost/" + this.envName + "/SAAS_BOOST_STACK"));
```

Comment:
Out of scope for this PR, but these `getExisting*FromSsm()` functions could be just compiled into one `getPropertyFromSsm(String ssmPropertyName, Object default, Class returnType)` to save some code duplication.

Reply:
SSM Parameter values are always String so we don't need a class argument.